pkg/envoy/cds/response.go

```diff
@@ -89,6 +99,10 @@ func NewResponse(meshCatalog catalog.MeshCataloger, proxy *envoy.Proxy, _ *xds_d
 		clusters = append(clusters, getTracingCluster(cfg))
 	}
 
+	return removeDups(proxy, clusters), nil
+}
+
+func removeDups(proxy *envoy.Proxy, clusters []*xds_cluster.Cluster) []types.Resource {
```

Comment:
needs a unit test.

Reply:
why is a proxy object needed here to dedup?

Reply:
logging the name to maintain the original logging statement. Changed to accept just the string and not the full object.

Reply:
This logging seems unnecessary, given it's deduping and clearly it's okay to have duplicates before dedup as per the current logic.

Reply:
done!